output.c

```diff
@@ -211,10 +211,7 @@ handle_output_damage_frame(struct wl_listener *listener, void *data)
 		goto damage_finish;
 	}
 
-	int output_width, output_height;
-	wlr_output_transformed_resolution(output->wlr_output, &output_width, &output_height);
-
-	wlr_renderer_begin(renderer, output_width, output_height);
+	wlr_renderer_begin(renderer, output->wlr_output->width, output->wlr_output->height);
```

Comment:
Why did you access the output width and height directly instead of through `wlr_output_transformed_resolution`?

Reply:
Apparently `wlr_renderer_begin` expects the real resolution. Rootston does it like this (https://github.com/swaywm/wlroots/blob/master/rootston/render.c#L240). Tinywl does not support output rotation so passing the transformed resolution did not make a difference, but I believe this was a bug.

Reply:
Indeed, thanks for catching this.